resources/views/livewire/quill-text-editor.blade.php

```diff
@@ -10,9 +10,27 @@
     
     quill.root.innerHTML = $wire.get('value');
     
-    quill.on('text-change', function () {
-        let value = quill.root.innerHTML;
-        @this.set('value', value);
+    let timeout = null;
+
+    function handleTextChange() {
+        if (timeout !== null) {
+            clearTimeout(timeout);
+        }
+
+        timeout = setTimeout(function() {
```

Comment:
I'm not sure if this is the best way to handle this.

Reply:
why??

Reply:
Because you are using a timeout.